### ISSUE TITLE
There is not dropout op or mask op in the encoder

### ISSUE THREAD

**Issue by zipzou** [princeton-nlp/SimCSE#92]
I reviewed the source code, but I can not find any dropout or additional mask operation in the code.
The encoder is implemented by `transformers`, and this model only added `Pooler` and `MLP` layers, and calculate the loss with `bmm`.

**So, how could you implement the dropout and mask introduced in the paper?**

What I can get is that you overwrite your own dataloader. While loading the data, copy the data to make the sentence pair, then masked the sentence based on the method from `BERT`, and the both of the sentences have different masked `input_ids`.

And I checked the `attention_mask`, it is same with `BERT` too.

**So, is the dropout or mask in the paper happened while the sentences masking generation?**

Could you answer this question? Thanks.

**Comment by BaohaoLiao** [comment]
I think the author uses the original BERT dropout. If you input the same sentence to BERT twice, they have different representation because of the original BERT dropout during training mode.

**Comment by gaotianyu1350** [comment]
Thanks @BaohaoLiao for the clarification! Hi @zipzou, we simply use the original Transformers dropout operation, and you can refer to our paper for details.
